src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -56,65 +59,103 @@ public void start(Stage primaryStage) throws Exception {
 
         // Sign in page
 
-        this.thestage = primaryStage;            // Initialize the main stage
+        this.thestage = primaryStage;                           // Initialize the main stage
+
+        BorderPane signInPane = new BorderPane();               // Initialize panes
```

Comment:
I would try to split the different screens into different windows, or at least into different functions.